gordo_components/client/client.py

```diff
@@ -203,6 +203,9 @@ def _endpoints_from_watchman(self, endpoint: str) -> typing.List[EndpointMetadat
                 resolution=data["endpoint-metadata"]["metadata"]["dataset"][
                     "resolution"
                 ],
+                model_offset=data["endpoint-metadata"]["metadata"]["model"][
+                    "model-offset"
```

Comment:
As a mild backwards-compability should we maybe have it at 0 if this is not set?

Reply:
Good catchy!